core/src/main/java/de/bwaldvogel/mongo/oplog/NoopOplog.java

```diff
@@ -0,0 +1,31 @@
+package de.bwaldvogel.mongo.oplog;
+
+import java.time.Clock;
+
+import de.bwaldvogel.mongo.bson.Document;
+import io.netty.channel.Channel;
+
+public final class NoopOplog extends AbstractOplog {
+
+    private static final NoopOplog INSTANCE = new NoopOplog(null);
```

Comment:
Probably you can still avoid nulls with [Clock.systemDefaultZone](https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#systemDefaultZone--)